edgexfoundry/app-service-configurable
# fix: Configure proper sample AES256 key in applicable profiles.

fixes #349

Signed-off-by: Leonard Goodell <leonard.goodell@intel.com>







**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/app-service-configurable/blob/main/.github/CONTRIBUTING.md

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] I have added unit tests for the new feature or bug fix (if not, why?) **N/A for ASC**
- [ ] I have opened a PR for the related docs change (if not, why?) **N/A - already documented in recent PR**

## Testing Instructions
1. Modify sample profile as follows
   - Set ExectionOrder to `ExecutionOrder = "Encrypt, HTTPExport"`
   - Configure HTTPExport MimeType  as `MimeType = "text/plain"`
   - Configure HTTPExport Url to a URL that you can monitor. Personal PostMan Mock servers work good for this.
   - Comment out the block of `[Writable.Pipeline.PerTopicPipelines]` to reduce debug log messages
2. Clone and build the branch for this PR
3. Run the non-secure EdgeX stack with Device Virtual
4. running run ASC
```
WRITABLE_LOGLEVEL=DEBUG ./app-service-configurable -p=sample
```
5. Verify no errors in logs 
6. Verify the REST endpoint received the encrypted data properly 

## New Dependency Instructions (If applicable)
**N/A**
